saleor/core/utils/__init__.py

```diff
@@ -89,3 +89,11 @@ def to_local_currency(price, currency):
             return exchange_currency(price, currency)
         except ValueError:
             pass
+
+
+def get_default_country(request):
```

Comment:
This name is misleading - I'd recommend to rename it to `get_user_shipping_country`